### ISSUE TITLE
q_posterior_logits returns log probabilities not logits

### ISSUE THREAD

**Issue by SizhuangHe** [cloneofsimo/d3pm#7]
Hi, thanks for your implementation for D3PM. 

I found in `D3PM.q_posterior_logits`, the output is actually log probabilities rather than logits. The `D3PM.vb` function essentially computes KL divergence from two logits which are returned by `D3PM.q_posterior_logits`. I'm not sure if this is a bug or I'm not understanding it correctly.

Hope to hear from you!
